packages/kubernetes/template/script.sh

```diff
@@ -125,6 +131,13 @@ function get_latest_sonobuoy_release_version() {
 }
 
 function update_available_versions() {
+    local version126=( $( for i in "${VERSIONS[@]}" ; do echo $i ; done | grep '^1.26' ) )
+    if [ ${#version126[@]} -gt 0 ]; then
+        if ! sed '0,/cron-kubernetes-update-126/d' ../../../web/src/installers/versions.js | sed '/\],/,$d' | grep -q "${version126[0]}" ; then
```

Comment:
I think you're missing the SED target for this line

Reply:
Done.